### DISCUSSION TITLE
清爽首页的频道入口问题

### DISCUSSION THREAD

**Discussion by pinaZC** [the1812/Bilibili-Evolved#3258] [功能建议 / Ideas]
很喜欢用清爽首页，布局和功能都很美观和实用，但是为什么没有原版首页上的频道入口呢？
原版的频道也是 动态、热门 并列的三大入口之一，它相当于只适合app端的话题功能，也算是网页端遗老找特定兴趣爱好的入口
![image](https://user-images.githubusercontent.com/72682556/163741265-24f90493-d5f8-445c-b8d3-a234c6bd61e1.png)

个人建议把现在的直播按钮换成频道，毕竟顶栏上已经有直播的入口了
![image](https://user-images.githubusercontent.com/72682556/163741059-2496b147-3e6c-44bc-b2f6-7a3d742c6bf3.png)

**Comment by Geeyun-JY3** [comment]
自定义顶栏是自定义顶栏，清爽首页是清爽首页，没啥关系的……而且自定义顶栏的入口能自定义，有些人也没开顶栏上的 `直播` 入口。

**Reply by pinaZC** [reply]
那就加个频道入口吧
